src/indicators/drought/biomasse.py

```diff
@@ -0,0 +1,335 @@
+"""Download and process Biomasse data
+
+The Biomasse data is produced by Action Against Hunger, Spain,
+and made available through the website http://geosahel.info/.
+Documentation of the methodology is made available through
+https://sigsahel.info/wp-content/uploads/2020/12/ACF_BioHydroGenerator_User_Guide.pdf.
+The Biomasse analysis uses Copernicus Global Land Service dry matter
+production (DMP) data: https://land.copernicus.eu/global/products/dmp.
+It is a dekadal product and is updated around 1 week to
+10 days after the end of the period.
+
+While the calculated cumulative biomasse and anomaly data
+is presented at a yearly level on the GeoSahel portal, it
+is not available in dekadal form. The only dekadal form data
+is for DMP values downsampled to the ADMIN0-2 levels across
+the region of interested. The code below accesses this dekadal
+data and then recalculates cumulative biomasse and anomaly
+using that. Since DMP measurements are productivity values, the
+calculation of total biomasse in a year is taken by averaging
+the DMP over the entire period and multiplying by 365.25.
```

Comment:
Aah this took me a while to get. But is it correct that the dekadal DMP is the mean daily production during that dekad? I first interpreted it as the total production during that dekad. But if the first is correct then the calculation of the total biomasse makes sense to me. If so, maybe you could clarify that here?

Reply:
Yeah, you ar ecorrect, have clarified in the documentation!